pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  freezed_annotation: ^2.0.1
+  freezed_annotation: ^3.0.0
```

Comment:
Note that I removed the dev_dependency of freezed but left the dependency. This is to avoid forcing us to support dart 3.6+ which is a much recent version of the Dart SDK. `freezed_annotation` does support Dart 3.0+.

We are planning to slowly move away from freezed, so we can remove this dependency as well in the future.